internal/server/handler.go

```diff
@@ -178,7 +195,7 @@ func (s *server) savePDVHandler(w http.ResponseWriter, r *http.Request) {
 
 // listPDVHandler lists pdv from storage.
 func (s *server) listPDVHandler(w http.ResponseWriter, r *http.Request) {
-	// swagger:operation GET /pdv/{owner} PDV List
+	// swagger:operation GET /v1/pdv/{owner} PDV List
```

Comment:
this way we need to remove baseUrl from the api's main definition, otherwise swagger ui will not provide access to api (will add additional /v1 prefix to requests)

Reply:
Thx, did not notice that, reverted.